app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -7,9 +7,28 @@ class VerificationRequest < Request
         private
 
         def build_request_body
+          if config.ddp_policy == IdentityConfig.store.lexisnexis_authentication_threatmetrix_policy
```

Comment:
I'd wonder if, rather than targeting specific policies, we could build the request body based on what's available in the `applicant` hash, assuming that the proofing result would fail if those attributes are required and not provided.

e.g.

```rb
def build_request_body
  request_body = {
    api_key: config.api_key,
    org_id: config.org_id,
    account_email: applicant[:email],
    event_type: 'ACCOUNT_CREATION',
    policy: config.ddp_policy,
    service_type: 'all',
    session_id: applicant[:threatmetrix_session_id],
    input_ip_address: applicant[:request_ip],
    account_address_street1: applicant[:address1],
    account_address_street2: applicant[:address2],
    account_address_city: applicant[:city],
    account_address_state: applicant[:state],
    # ...
  }.compact

  if request_body[:account_address_street1].present?
    request_body = {
      account_address_street2: '',
      **request_body,
      account_address_country: 'US',
    }
  end

  # ...etc

  request_body
end
```